### PR TITLE
Docs/user docs

### PR DESCRIPTION
**Pull request by pavel-collab**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,48 @@
...
+![График использования памяти](/docs/images/memory_usage_example.png)
+
+![График использования памяти](/docs/images/home_page_plate_example.png)
+
+![График использования памяти](/docs/images/cpu_usage_example.png)
+
+## Запуск демона
+
+Для начала работы приложения убедитесь, что на вашем хосте установлены необходимые зависимости, их вы можете посмотреть в файле _requirements.txt_. 
+Если у вас не хватает каких-либо зависимостей, вы можете их доустановить или создать для приложение отдельное виртуальное окружение:
```
Comment (barannikovav) [inline README.md:22]: для приложениЯ

File: README.md [hunk 2]
```diff
@@ -1,8 +1,48 @@
...
+```
+python3 -m venv ./.venv
+source ./.venv/bin/activate
+pip install -r requirements.txt
+```
+
+Перед запуском основной программы убедитесь, что у вас настроет сервер postgresql. Вы можете прочитать инструкции по установке и настройке
```
Comment (barannikovav) [inline README.md:29]: настроен

File: README.md [hunk 3]
```diff
@@ -1,8 +1,48 @@
...
+PostgreSQL и TimescaleDB в каталоге __docs__ или обратитесь к data base administrator вашей компании. Инструкции по настройке кластера 
+postgresql вы можете найти в файле [postgresql.md](/docs/postgres.md), а по настройке базы данных в [timescale.md](/docs/timescaledb.md).
+В конфигурационном файле _conf/connection.json_ заполните обязательные поля:
+- _dbname_ -- имя вашей базы данных
+- _host_ip_ -- ip-адрес вашего postgresql кластера (если кластер развернут на локальном хосте, можно указать _localhost_)
+
+Если все зависимости установлены, вы можете просто запустить скрипт _main.py_ на том хосте, где зарущены ваши докер-контейнеры
```
Comment (barannikovav) [inline README.md:36]: заПущены

File: docs/database_description.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+содержит основную информацию о контейнерах: id, имя контейнера, статус активности и ограничения по памяти. Оставшиеся таблицы содержат 
+статистическую информацию об этих контейнерах, разбитую на логические группы:
+- __container_stats__ -- общая статистика об использовании памяти и процессора
+- __container_detailed_statistics__ -- более детальная статистика об использовании процессора
+- __container_networks__ -- статистика об использовании сети
+
+![Граф зависимостей таблиц](/docs/images/relationships.implied.large.png)
+
+Кроме таблиц есть представления:
+- __dashboard_table__ -- основная агригированная информация по всем контейнерам; значения из этого представления используются для отображения инфомарции на заглавной страницы веб-интерфейса
```
Comment (barannikovav) [inline docs/database_description.md:15]: агрЕгированная, инфоРмации, страницЕ

File: docs/database_description.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+- __container_aggregation__ -- более расширенная агрегированная статистика, которая расчитывается каждый час
+
+![Представление dashboard](/docs/images/dashboard_table.1degree.png)
+
+![Представление aggregation](/docs/images/container_aggregation.1degree.png)
+
+Таблицы со статистиками и представления связаны с основной таблицей __container_info__ через внешние ключи. В основной таблице идегтификатор
```
Comment (barannikovav) [inline docs/database_description.md:22]: идеНтификатор

**Review by barannikovav — CHANGES_REQUESTED** [review]
Fix missspells

File: README.md (hunk 1 above)
Comment (pavel-collab) [reply README.md:22]: resolved

File: README.md (hunk 2 above)
Comment (pavel-collab) [reply README.md:29]: resolved

File: README.md (hunk 3 above)
Comment (pavel-collab) [reply README.md:36]: resolved

File: docs/database_description.md (hunk 4 above)
Comment (pavel-collab) [reply docs/database_description.md:15]: resolved

File: docs/database_description.md (hunk 5 above)
Comment (pavel-collab) [reply docs/database_description.md]: resolved

**Comment by pavel-collab** [comment]
All minor comments are fixed

**Review by barannikovav — APPROVED** [review]
All my comments have been revised in this version.

**Pull request merged**